tensorflow/python/ops/nn_ops.py

```diff
@@ -5181,7 +5181,7 @@ def dropout_v2(x, rate, noise_shape=None, seed=None, name=None):
     if build_info.build_info['is_rocm_build'] and \
        (x.dtype == dtypes.float64 or x.dtype == dtypes.float32 \
         or x.dtype == dtypes.float16) and def_dropout != "1" \
-        and null_noise_shape:
+        and is_executing_eagerly and null_noise_shape:
```

Comment:
are you sure that `is_executing_eagerly` == XLA enabled?

Reply:
it probably isn't.  just got a GPU unit test failure...